docs/.vitepress/config.mts

```diff
@@ -34,6 +32,12 @@ export default withMermaid({
                     text: 'Getting Started', link: `${PATH_TECHNIK}get_started/`
                 },
                 {text: 'Entwicklungsumgebung', link: `${PATH_TECHNIK}development/`},
+                {
+                    text: 'Guides', link: `${PATH_GUIDES}`, collapsed: false, items: [
```

Comment:
```suggestion
                    text: 'Guides', link: `${PATH_GUIDES}`, collapsed: true, items: [
```

Reply:
siehe commit [a109f47](https://github.com/it-at-m/Wahllokalsystem/pull/495/commits/a109f478e0698f1fe97a33e8dd92fba7178a6011)